### ISSUE TITLE
exit() builtin method is missing

### ISSUE THREAD

**Issue by PragTob** [shoes/shoes4#894]
We are still missing the builtin `#exit` method. See [builtin methods](http://shoesrb.com/manual/Built-in.html)

It says:

```
**exit()**
Stops your program. Call this anytime you want to suddenly call it quits.

PLEASE NOTE: If you need to use Ruby's own exit method (like in a forked Ruby process,) call Kernel.exit.
```

I guess we could just iterate over all registered Shoes apps and call `quit` ?

Should not be a hard issue to tackle.

**Comment by ducki13** [comment]
What is the simplest way to obtain all running apps? I found an @apps variable in shoes-core/lib/shoes/common/registration.rb, should I use it?

**Comment by jasonrclark** [comment]
Yeah, the `@apps` instance variable in the `Shoes` module itself is the master list of apps. If you need at it from outside of `Shoes` or the `Common::Registration` module, there is an accessor `Shoes.apps` which can be used.

**Comment by PragTob** [comment]
This is already tackled in #1129

**Comment by jasonrclark** [comment]
That PR didn't get merged, and with #1474 we essentially our Shoes 4 implementation of `quit` is the same as `exit`
